src/lib/get-marked-with-highlighter.ts

```diff
@@ -12,5 +16,6 @@ export const getMarked = (options: marked.MarkedOptions, extensions: marked.Mark
 		...options,
 	});
 	marked.use(...extensions);
+	marked.use({ extensions: tokenizerExtensions });
```

Comment:
I think the current `marked_extensions` might be wrong, should probably fix that, instead of adding an extra option?

Reply:
Based on [documentation](https://marked.js.org/using_pro#extensions), it seems that extensions should be passed like this `{extensions: [..]}`
I updated typing and it seems to work like that